packages/x-date-pickers/src/DateCalendar/DateCalendar.tsx

```diff
@@ -88,10 +88,11 @@ type DateCalendarComponent = (<TDate>(
 ) => React.JSX.Element) & { propTypes?: any };
 
 /**
- *
  * Demos:
  *
- * - [Date Picker](https://mui.com/x/react-date-pickers/date-picker/)
+ * - [DatePicker](https://mui.com/x/react-date-pickers/date-picker/)
```

Comment:
Could we add `DateCalendar` to the `date-picker.md` `components` lists to make this correct? 🤔 
We could also add `PickersDay` to align more with the `date-range-picker.md`. 🤔